src/components/modal.jsx

```
@@ -34,13 +34,13 @@
   const renderCatImages = () => {
     
     return (
-    <div className="modal--carrousel">
+    <div className="flex">
       {images.map((image, index) => (
-        <div key={index} className="modal--image-container">
+        <div key={index} className="w-36 h-36 overflow-hidden mr-1 rounded">
           <img 
             src={image.url} 
             alt="Cat Image" 
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)